packages/admin-next/dashboard/src/routes/orders/order-create-claim/claim-create.tsx

```diff
@@ -0,0 +1,91 @@
+import { useEffect, useMemo, useState } from "react"
+import { useTranslation } from "react-i18next"
+import { useNavigate, useParams } from "react-router-dom"
+
+import { toast } from "@medusajs/ui"
+
+import { RouteFocusModal } from "../../../components/modals"
+import { ClaimCreateForm } from "./components/claim-create-form"
+
+import { useOrder, useOrderPreview } from "../../../hooks/api/orders"
+import { useClaims, useCreateClaim } from "../../../hooks/api/claims"
+import { DEFAULT_FIELDS } from "../order-detail/constants"
+
+let IS_REQUEST_RUNNING = false
+
+export const ClaimCreate = () => {
+  const { id } = useParams()
+  const navigate = useNavigate()
+  const { t } = useTranslation()
+
+  const { order } = useOrder(id!, {
+    fields: DEFAULT_FIELDS,
+  })
+
+  const { order: preview } = useOrderPreview(id!)
+
+  const [activeClaimId, setActiveClaimId] = useState()
+
+  const { mutateAsync: createClaim } = useCreateClaim(order.id)
+
+  // TODO: GET /claims/:id is not implemented
+  // const { claim } = useClaim(activeClaimId, undefined, {
+  //   enabled: !!activeClaimId,
+  // })
```

Comment:
todo: This was introduced yesterday. Maybe just replace it in a follow-up PR

Reply:
Its already replaced [here](https://github.com/medusajs/medusa/pull/8480/files#diff-9c80b20f3219a9609f9c97f04f2c81a53858a69c5c8ce700b9df73e6381e0d3dR29)